samples/message-proposal-1.html

```diff
@@ -13,7 +13,7 @@
 <meta name="Subject" content="Streams after receiving GOAWAY" />
 <meta name="Date" content="2015-11-15" />
 
-<link href="https://www.w3.org/scripts/bootstrap/3.3/css/bootstrap.css"
+<link href="//www.w3.org/scripts/bootstrap/3.3/css/bootstrap.min.css"
```

Comment:
There is no need to ever use anything else than the HTTPS version of it. This said, protocol relative paths break in local testing as browsers resolve it to the filesystem.